charts/postcoder-proxy/values.yaml

```diff
@@ -13,15 +13,16 @@ externalSecrets: &external_secrets_config
   stores: {}
 
 env:
-  APP_ENV: production
+  APP_ENV: development
+  NEW_RELIC_AGENT_ENABLED: 'false'
   CACHE_TTL: '2592000'
   CACHE_URL: redis://redis-master:6379/1
 
 image:
-  repository: 979633842206.dkr.ecr.eu-west-1.amazonaws.com/postcoder
+  repository: public.ecr.aws/citizensadvice/postcoder
   pullPolicy: IfNotPresent
   # tag defaults to the chart app version
-  # tag: v2.0.0
+  tag: main
```

Comment:
It's because the new CI workflow for that project which is based on [this confluence page](https://citizensadvice.atlassian.net/wiki/spaces/OPS/pages/2552856599/GitHub+actions#Publish-PUBLIC-docker-image-to-CA's-PUBLIC-Container-Registry) the tags that it assigns to the images are `branch` and `sha-<short commit sha>, so this change is necessary for accessing the public image